onboarding/mentors.md

```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
+
+## Why
+One of Artsy's core values is that [People are Paramount](#). Artsy's engineering team works to reflect this in practice...
+
+## What makes a good mentor (Keep?)
+- A first stop for stupid questions
+
+## Mentorship from day 0
+- Keep tabs on your mentee's schedule. Their first day will be full of meetings with People Ops but some good tasks for a mentor[:
+  - Introduce yourself
+  - Set a time for an [introductory coffee or tea](#your-first-meeting)
+  - Get in touch with the mentee's manager
+  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
+    - Point people in Workplace & People Ops
+    - Their manager
+    - Their team lead
+    - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
+  
+
+### Your first meeting
+Get to know each other. This can be a nice, casual conversation. Focus on active listening.
+- Talk about your own history at and before joining Artsy
+- Where are they coming from?
+  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote work?
+  - Are they new in town? Where do they live, for how long? Just have a normal conversation.
+- Prior experience with our technology stacks (and others)
+- Other relevant past experience
+- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+### Your first week
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
```

Comment:
Worth noting that the mentor should encourage the new hire to update  anything that is out-of-date 😄